1001-2000/LIT1804Lifeof.xml

```diff
@@ -52,6 +52,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
       </encodingDesc>
       <profileDesc>
          <langUsage><language ident="en">English</language><language ident="gez">Gǝʿǝz</language></langUsage>
+         <abstract>
+<p>According to the colophon in <ref type="mss" corresp="EMML1840">EMML 1840, fol. 223r</ref> the text was commissioned by the <roleName type="title">ʿaqqābe saʿat</roleName> <ref type="pers" corresp="PRS8538saraqaB">Śaraqa Bǝrhān</ref>.</p>
```

Comment:
I am not sure fol. 223r can be nested here inside `ref type="mss"`, Is this marking up used elsewhere or allowed in the guidelines?

Reply:
I think, I have pointed to specific folia in manuscripts like this in other records, as well. I think, it will work.